File: Java/1347_Minimum_Number_of_Steps_to_Make_Two_Strings_Anagram.java
class Solution {
    public int minSteps(String s, String t) {
        int[] st = new int[26];
        int[] tt = new int[26];
        for(int i = 0; i < s.length(); i++){
            st[s.charAt(i) - 'a']++;
            tt[t.charAt(i) - 'a']++;
        }
        int result = 0;
        for(int i = 0; i < 26; i++){
            result += Math.abs(st[i] - tt[i]);
        }
        return result/2;
    }
}

File: CS/1347_Minimum_Number_of_Steps_to_Make_Two_Strings_Anagram.cs
public class Solution {
    public int MinSteps(string s, string t) {
        int[] p = new int[26];
        int result = 0;
        for(int i = 0; i < s.Length; i++){
            p[s[i] - 'a']++;
            p[t[i] - 'a']--;
        }
        return p.Where(x => x > 0).Sum();
        /*
        for(int i = 0; i < 26; i++){
            if(p[i] > 0)
                result += p[i];
        }
        return result;
        */
    }
}
